src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -99,25 +108,30 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
     function initialize(
         address _churnApprover,
         address _ejector,
-        OperatorSetParam[] memory _operatorSetParams,
         IPauserRegistry _pauserRegistry,
-        uint256 _initialPausedStatus
+        uint256 _initialPausedStatus,
+        OperatorSetParam[] memory _operatorSetParams,
+        uint96[] memory _minimumStakes,
+        IStakeRegistry.StrategyAndWeightingMultiplier[][] memory _strategyParams
```

Comment:
should this be called `_initialQuorumParams` or something?

Reply:
ehh actually the other things are related enough to quorums, I think it makes at least as much sense as-is.